Dockerfile

```diff
@@ -39,3 +31,6 @@ WORKDIR /luigi
 # gets populated. Additionally, /luigi/tasks is where we expect python code to
 # be mounted.
 ENV PYTHONPATH /luigi/tasks/qgreenland:/opt/conda/share/qgis/python/plugins:/opt/conda/share/qgis/python
+ENV PATH /opt/conda/bin:$PATH
+
+CMD ["/usr/local/bin/luigid"]
```

Comment:
I tried adding a third "distroless" or alpine stage to the build, but had some mysterious issues with executing files.